app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, human_age",
+        [
+            (-1, -2, [0, 0]),
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+        ],
+        ids=[
+            "Numbers should be positive",
```

Comment:
Use more informative descriptions here: e.g. `negative cat and dog age must equal 0 in human age`